app/models/letter.rb

```diff
@@ -29,5 +29,23 @@ def letter_date
   def template_name
     @letter_template&.name
   end
+
+  def letter_recipient
+    case @letter_template.template_type
+    when "dispatch"
+      values.recipient_name
+    when "acknowledgement"
+      values.requester_name
```

Comment:
This is a hard one for naming. From the perspective of the models you'd always think that `recipient_name` is was who the letter was going to - but that's not the case in reality.

Thinking whether there is a level of abstraction we are missing that could make this clearer.

Reply:
Agreed  - I think when we get the next type of letter (probably chasing) we could look at refactoring the letter stuff